manifest_staging/charts/gatekeeper/templates/gatekeeper-mutating-webhook-configuration-mutatingwebhookconfiguration.yaml

```diff
@@ -43,7 +43,7 @@ webhooks:
       - {{ . }}
       {{- end }}
     {{- end }}
-  objectSelector: {{ toYaml .Values.mutatingWebhookObjectSelector }}
+  objectSelector: {{ toYaml .Values.mutatingWebhookObjectSelector | nindent 4 }}
```

Comment:
Can you pls update https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/replacements.go#L123 instead and then run `make manifests` to generate the yaml updates in manifest_staging/

Reply:
followed the correct way of modifying chart. thank you @ritazh :)